sentry-spring/src/test/kotlin/io/sentry/spring/SentryUserProviderEventProcessorTest.kt

```diff
@@ -200,4 +206,13 @@ class SentryUserProviderEventProcessorTest {
             assertNull(it.ipAddress)
         }
     }
+
+    @Test
+    fun `Transaction sets the user`() {
```

Comment:
Either the name of this test is off or the test itself is not finished, as it does not seem that it is testing if user is set on the transaction. Also maybe `user is set on transaction` rather than `transaction sets the user`?

Reply:
good catch